src/gwt/src/org/rstudio/studio/client/workbench/AddinsMRUList.java

```diff
@@ -1,303 +0,0 @@
-import org.rstudio.studio.client.workbench.addins.Addins.RAddin;
-import org.rstudio.studio.client.workbench.addins.Addins.RAddins;
-import org.rstudio.studio.client.workbench.addins.AddinsCommandManager;
-import org.rstudio.studio.client.workbench.addins.AddinsKeyBindingsChangedEvent;
-import org.rstudio.studio.client.workbench.addins.events.AddinRegistryUpdatedEvent;
-import org.rstudio.studio.client.workbench.commands.Commands;
-import org.rstudio.studio.client.workbench.events.ListChangedEvent;
-import org.rstudio.studio.client.workbench.events.ListChangedHandler;
-import org.rstudio.studio.client.workbench.events.SessionInitEvent;
-import org.rstudio.studio.client.workbench.events.SessionInitHandler;
-import org.rstudio.studio.client.workbench.model.Session;
-
-import com.google.inject.Inject;
-import com.google.inject.Provider;
-import com.google.inject.Singleton;
-
-@Singleton
-public class AddinsMRUList implements SessionInitHandler,
-                                      AddinRegistryUpdatedEvent.Handler,
-                                      AddinsKeyBindingsChangedEvent.Handler
-{
-   @Inject
-   public AddinsMRUList(Provider<WorkbenchListManager> pListManager,
-                        Provider<AddinsCommandManager> pAddinManager,
-                        Session session,
-                        EventBus events,
-                        Commands commands)
-   {
-      pListManager_ = pListManager;
-      pAddinManager_ = pAddinManager;
-      session_ = session;
-      events_ = events;
-      commands_ = commands;
-      
-      mruCommands_ = new AppCommand[] {
-            commands_.addinsMru0(),
-            commands_.addinsMru1(),
-            commands_.addinsMru2(),
-            commands_.addinsMru3(),
-            commands_.addinsMru4(),
-            commands_.addinsMru5(),
-            commands_.addinsMru6(),
-            commands_.addinsMru7(),
-            commands_.addinsMru8(),
-            commands_.addinsMru9(),
-            commands_.addinsMru10(),
-            commands_.addinsMru11(),
-            commands_.addinsMru12(),
-            commands_.addinsMru13(),
-            commands_.addinsMru14()
-      };
-      
-      events_.addHandler(SessionInitEvent.TYPE, this);
-      events_.addHandler(AddinRegistryUpdatedEvent.TYPE, this);
-      events_.addHandler(AddinsKeyBindingsChangedEvent.TYPE, this);
-      
-      initCommandHandlers();
-   }
-   
-   private void initCommandHandlers()
-   {
-      for (int i = 0; i < mruCommands_.length; i++)
-         addIndexedHandler(mruCommands_[i], i);
-   }
-   
-   private void addIndexedHandler(AppCommand command, int index)
-   {
-      command.addHandler(new AddinCommandHandler(index));
-   }
-
-   @Override
-   public void onSessionInit(SessionInitEvent sie)
-   {
-      RAddins addins = session_.getSessionInfo().getAddins();
-      update(addins);
-      
-      mruList_ = pListManager_.get().getAddinsMruList();
```

Comment:
Should we also remove the code backing this list? https://github.com/rstudio/rstudio/blob/54cd3abcfc58837b433464c793fe9b03a87f0bb4/src/gwt/src/org/rstudio/studio/client/workbench/WorkbenchListManager.java#L101